pyproject.toml

```diff
@@ -1,3 +1,25 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "kyber-py"
+version = "1.0.1"
+requires-python = ">=3.9"
+description = "A pure python implementation of ML-KEM (FIPS 203)"
+readme = "README.md"
+classifiers = [
+    "Topic :: Security :: Cryptography",
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
```

Comment:
PEP 639 deprecated this in favor of `License-Expression` in the core packaging metadata.
```suggestion
```